<launch>
  <!-- Lane detector nodelet

  Subsribes to topic "image_rect_mono"
  -->
  <node
    pkg="ghost" 
    type="lane_detector_node" 
    name="lane_detector" 
    output="screen">
    <remap from="image_rect_mono" to="camera/color/image_rect"/>
  </node>
  
  <!-- Load the parameters -->
  <rosparam 
      file="$(find ghost)/param/lane_detector_params.yaml" 
      command="load"/>

</launch>